### ISSUE TITLE
column level seems to be out of bounds here

### ISSUE THREAD

**Issue by pgbovine** [SamLau95/pandas_tutor#126]
https://github.com/SamLau95/pandas_tutor/blob/4ad1a2934298a1ff2d783e796bd61e50cd4c9c0e/pandas_tutor/tests/e2e_golden/unstack_series_01.py.golden#L30

this seems like it should be level 0 and not 1, right?

**Comment by pgbovine** [comment]
it seems like these tests may also have this bug (they also fail the same asserts in my canonicalization script):

```
ptable_only_columns_01.py
stack_basic_01.py
stack_into_unstack_01.py
unstack_series_01.py
unstack_series_02.py
reset_index_basic.py # <-- row level out of bounds
```

**Comment by SamLau95** [comment]
yeah, these are bugs on my end

**Comment by SamLau95** [comment]
fixed in 642568c4729afb38b9da4c62f2e0b185864698b1

**Comment by SamLau95** [comment]
just kidding, this is still breaking:

stack_into_unstack_01.py

**Comment by SamLau95** [comment]
fixed in 92fb98a121bd59ada73d692f260a1f636a76776e
